crm/modules/member/command.inc.php

```diff
@@ -121,6 +128,30 @@ function command_member_add () {
 }
 
 /**
+ * Handle member edit request.
+ *
+ * @return The url to display when complete.
+ */
+function command_member_edit () {
+    global $esc_post;
+    
+    $member_data = crm_get_data('member', array('cid'=>$_POST['cid']));
+    $member = $member_data[0]['member'];
+    $esc_cid = mysqli_real_escape_string($db_connect, $_POST['cid']);
```

Comment:
Should this esc_cid get set up at the top of this scope then used throughout instead of $_POST['cid'] ??

Reply:
Probably! this code has been sitting around for over a year so I'd need to tweak & test it again!